### PR TITLE
Minor log optimizations to improve readability and filtering in grafana

### PR DESCRIPTION
**Pull request by bonomat**

### CODE REVIEW

**Review by luckysori — APPROVED** [review]
Very nice, thanks!

File: daemon/src/connection.rs [hunk 1]
```diff
@@ -429,7 +429,7 @@ impl Actor {
 
         let msg_name = msg.name();
 
-        tracing::trace!(target: "wire", "Received {msg_name}", );
+        tracing::trace!(target: "wire", msg_name = msg_name, "Received");
```
Comment (klochowicz) [inline daemon/src/connection.rs]: nit: `msg_name` probably doesn't need to be repeated, as we're not renaming it here

**Review by klochowicz — APPROVED** [review]
nice cleanup! 👍

**Pull request merged**